packages/editor/src/components/editor/FormToolbar.tsx

```diff
@@ -67,6 +76,31 @@ export const FormToolbar = () => {
           </ToolbarContainer>
         )}
       </Flex>
+
+      <Flex gap={4} className='palette-section'>
+        <PaletteCategoryPopover
+          label='Structure'
+          icon={IvyIcons.LaneSwimlanes}
+          items={componentsByCategoryGroupBySubcategory('Structure')}
```

Comment:
why set the items as props?
You have the label (category) so you can simply setup the items in the PaletteCategoryPopover itself...